designer/submodules/packages/components/src/common/nuIcon.tsx

```diff
@@ -1,8 +1,20 @@
 import React from "react";
 import loadable from "@loadable/component";
 import { styled } from "@mui/material";
+import { CrackedNut } from "./CrackedNut";
 
-const Icon = loadable(async () => import("nussknackerUi/Icon"));
+// This retry mechanism was introduced as sometimes dynamic import would fail.
+const Icon = loadable(() =>
+    import("nussknackerUi/Icon").catch(async (error) => {
+        console.warn("Failed to import Icon submodule. Import retry will be carried out", error);
+        try {
+            return await import("nussknackerUi/Icon");
+        } catch (e) {
+            console.error("Import retry failed", e);
+            return { default: () => <CrackedNut /> };
```

Comment:
This cracked nut isn't good UX stuff :( Maybe we can show user something more friendly?

Reply:
As i mentioned these changes aim to gather some more information about the root of the issue and check whether simple retries would help. If they don't then we need some kind of a signal for the user on the screen so they would check the logs that we could analyze. This cracked nut is not as flashy as in some places. It will be displayed in the place of the problematic icon and the app would still be usable.